<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XProcessDialog</class>
 <widget class="QDialog" name="XProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>265</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QRadioButton" name="radio_graph">
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
	color: rgb(64, 65, 66);
	font: bold 14px;
 }
</string>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Y min =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="line_y_min">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X max =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="line_x_expression">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="line_x_max">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="line_y_max">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Y max =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="line_x_min">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radio_expression">
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
	color: rgb(64, 65, 66);
	font: bold 14px;
 }
</string>
      </property>
      <property name="text">
       <string>Expression</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X min =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="calculate">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(236, 236, 236);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(64, 65, 66);
    color: rgb(64, 65, 66);
    font: bold 14px;
    padding: 6px;
}

QPushButton:pressed{
    background-color: rgb(140, 140, 140);
}</string>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
